Ryujinx.HLE/Utilities/UInt128Utils.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+
+namespace Ryujinx.HLE.Utilities
+{
+    static class UInt128Utils
+    {
+        public static UInt128 FromHex(string hex)
+        {
+            return new UInt128((ulong)Convert.ToInt64(hex.Substring(16), 16), (ulong)Convert.ToInt64(hex.Substring(0, 16), 16));
```

Comment:
```suggestion
            return new UInt128((ulong)long.Parse(hex.AsSpan(16), 16), (ulong)long.Parse(hex.AsSpan(0, 16), 16));
```
Avoids the Substring allocations

Reply:
Nevermind, didn't see the second parameter.